docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -348,6 +348,313 @@ and a file system location of ``/var/my_tables/test_table``::
+     hdfs://hadoop-master:9000/user/hive/warehouse/test_table/metadata/faa19903-1455-4bb8-855a-61a1bbafbaa7-m0.avro |  6277           |   0                  | 7860805980949777961   |  1                       |   0                         |  0                          |{{contains_null=false, lower_bound=1, upper_bound=1},{contains_null=false, lower_bound=2021-01-12, upper_bound=2021-01-12}}
+
+
+The output of the query has the following columns:
+
+.. list-table:: Manifests columns
+  :widths: 30, 30, 40
+  :header-rows: 1
+
+  * - Name
+    - Type
+    - Description
+  * - ``path``
+    - ``varchar``
+    - The manifest file location
+  * - ``length``
+    - ``bigint``
+    - The manifest file length
+  * - ``partition_spec_id``
+    - ``integer``
+    - The identifier for the partition
+  * - ``added_snapshot_id``
+    - ``bigint``
+    - The identifier of the snapshot during which this manifest entry has been added
+  * - ``added_data_files_count``
+    - ``integer``
+    - The number of data files added during the snapshot
+  * - ``deleted_data_files_count``
+    - ``integer``
+    - The number of data files deleted during the snapshot
+  * - ``partitions``
+    - ``array(row(contains_null boolean, lower_bound varchar, upper_bound varchar))``
+    - Partition range metadata
+
+
+``$partitions`` table
+^^^^^^^^^^^^^^^^^^^^^
+
+The ``$partitions`` table provides a detailed overview of the partitions
+of the  Iceberg table.
+
+You can retrieve the information about the partitions of the Iceberg table
+``test_table`` by using the query::
+
+    SELECT * FROM "test_table$partitions"
+
+.. code-block:: text
+
+     partition             | record_count  | file_count    | total_size    |  data
+    -----------------------+---------------+---------------+---------------+--------------------------------------
+    {c1=1, c2=2021-01-12}  |  2            | 2             |  884          | {c3={min=1.0, max=2.0, null_count=0}}
+    {c1=1, c2=2021-01-13}  |  1            | 1             |  442          | {c3={min=1.0, max=1.0, null_count=0}}
+
+
+The output of the query has the following columns:
+
+.. list-table:: Partitions columns
+  :widths: 20, 30, 50
+  :header-rows: 1
+
+  * - Name
+    - Type
+    - Description
+  * - ``partition``
+    - ``row(...)``
+    - A row which contains the mapping of the partition column name(s) to the partition column value(s)
+  * - ``record_count``
+    - ``bigint``
+    - The number of records under the partition
+  * - ``file_count``
+    - ``bigint``
+    - The number of files mapped under the partition
+  * - ``total_size``
+    - ``bigint``
+    - The size of all the files under the partition
+  * - ``data``
+    - ``row(... row (min ..., max ... , null_count bigint))``
+    - Partition range metadata
+
+``$files`` table
+^^^^^^^^^^^^^^^^
+
+The ``$files`` table provides a detailed overview of the data files corresponding to the  Iceberg table.
```

Comment:
All files or files active in current snapshot. If the latter please be more specific in desc.